### ISSUE TITLE
User suggestion: tape measure units, velocity of objects

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/gravity-and-orbits#268]
> I teach high school physics in Los Angeles and I truly appreciate the PhET simulations.
> 
> I want to comment on the "Gravity and Orbits" simulation.
> 
> 1)   I feel that the distance units of "miles" that the "tape measure" tool uses is not appropriate for real physics.  Is there a way to change the units?   If not, I suggest that there be a way to change the units to meters and to Astronomical Units.
> 
> 2)  I need a way to see and/or set the velocity of the objects.  Could we add a display box that shows the velocities in meters per second.  It would be nice if we could type into that box to set the initial velocity numerically?

**Comment by arouinfar** [comment]
> I feel that the distance units of "miles" that the "tape measure" tool uses is not appropriate for real physics. Is there a way to change the units? If not, I suggest that there be a way to change the units to meters and to Astronomical Units.

Miles is a bit of an odd (and USA-centric) choice. My best guess is that miles is better understood by middle school students in the USA, so that's why we chose that unit. That said, km or AU are also appropriate units, and would be a better choice for globalization or even HS/university students. I see two ways we could address this request.
1. Add 3 horizontal radio options below the Measuring Tape checkbox for mi, km, and AU. Things are already quite tight on the To Scale screen, but with some pixel polishing we could make enough space.
2. Set the units (mi, km, AU) in the Options menu, and perhaps add a query parameter to choose the default unit.

>I need a way to see and/or set the velocity of the objects. Could we add a display box that shows the velocities in meters per second. 

On the To Scale screen, we could replace the "Mass" checkbox with "Values" which would display the masses, velocity values, and possibly the value of the gravitational force too. The labels could look similar to what was done in Vector Addition, but with a semi-transparent black background behind the text.
![image](https://user-images.githubusercontent.com/8419308/66956598-e5b5e180-f021-11e9-82c8-8f4b02c33a56.png)

> It would be nice if we could type into that box to set the initial velocity numerically?

This is something that can be done in My Solar System, and I'd imagine its HTML5 design would behave similarly. I'd say this is beyond the scope of GAO.

> do you want to work on either of these recommendations before next publication?

These might be nice to address in the next version of GAO, if there's enough bandwidth for it. @ariel-phet what do you think about these suggestions?

**Comment by ariel-phet** [comment]
@arouinfar a couple of thoughts

1. For the tape measure, I think we should just change to "km" - it seems very odd to have this one in miles. I am guessing that is a hold over from Java. If we put the units in "thousand kms" I would be fine if we had an option menu to change to miles or AU, but it seems like km should be the default

2. If we do values of the velocity, they probably need to be in "km/s" or they will be very large numbers. I would be fine with changing to a "values" checkbox.

3. I agree, even if we show the value of the velocity, setting it numerically (with a number pad or such) is beyond the scope of this sim.

**Comment by arouinfar** [comment]
@kathy-phet and I talked about this, and we'd like to move forward with changing the measuring tape units to kilometers. This was also requested by another user in #323. In the future may consider a query parameter or options menu control to change the units, but we'll wait and see if it becomes a user request.

There is a (potential?) client who is interested in this sim who has some goals related to the velocity. Kathy suggested we that we wait on making any changes to the velocity representation until we have further discussions with them.

@samreid please make the following changes
- [ ] MeasuringTapeNode should display the distance in km
- [ ] Remove "thousand" from the readout. (Display `100000` instead of `100 thousand`.)
- [ ] Spell out "kilometers".

Default lengths:
- [ ] Sun/Earth & Sun/Earth/Moon: 80000000 kilometers
- [ ] Earth/Moon: 150000 kilometers
- [ ] Earth/Satellite: 3000 kilometers

**Comment by samreid** [comment]
I completed the changes above, @arouinfar can you please test it out?

**Comment by arouinfar** [comment]
@samreid looks good, but let's drop the decimal place from the Planet/Satellite scene.

![image](https://user-images.githubusercontent.com/8419308/82240840-5a266b00-98f8-11ea-90ed-ba9a68c9bd0d.png)

**Comment by samreid** [comment]
I removed it in the commit, please re-review.

**Comment by arouinfar** [comment]
Perfect, thanks @samreid!

I am going to mark this issue as deferred since there are still discussions to have about the velocity representation.
